vfs.go

```diff
@@ -396,3 +400,58 @@ func (v *VFS) DirSize(uri string) (uint64, error) {
 
 	return uint64(cfsize), nil
 }
+
+// Callback is a type
+type Callback struct {
+	Func func(string, unsafe.Pointer)
+	Path string
+	Data unsafe.Pointer
+}
+
+//export numOfFragmentsInPath
+func numOfFragmentsInPath(path *C.cchar_t, data unsafe.Pointer) int32 {
+	ctx, err := NewContext(nil)
```

Comment:
vfs needs to come from data pointer

Reply:
You should also pass in the config somehow because if the user sets credentials or configs those won't be set here.

Reply:
Provided that vfs comes from the caller, they will have the opportunity to set config when creating vfs object right?